biblio/bibliography.bib

```diff
@@ -1279,3 +1279,15 @@ @inproceedings{dtmfiltrations
   bibsource = {dblp computer science bibliography, https://dblp.org}
 }
 
+@unpublished{edgecollapsesocg2020,
```

Comment:
It is published: https://drops.dagstuhl.de/opus/volltexte/2020/12177/ provides a bibtex entry.

Reply:
I changed it on f7876ea08e810c57f90e0233fffbd91d57f6d037